gateway/transport/plugins/jira/jira.go

```diff
@@ -0,0 +1,173 @@
+package jira
```

Comment:
Adding code to inside existing plugin structure is something quite fine while we don't migrate it, but we can not create new plugins. We should find a new place to all of this